homeassistant/components/upnp/device.py

```diff
@@ -9,16 +10,26 @@
 
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.typing import HomeAssistantType
+import homeassistant.util.dt as dt_util
 
-from .const import CONF_LOCAL_IP, DOMAIN, LOGGER as _LOGGER
+from .const import (
+    BYTES_RECEIVED,
+    BYTES_SENT,
+    CONF_LOCAL_IP,
+    DOMAIN,
+    LOGGER as _LOGGER,
+    PACKETS_RECEIVED,
+    PACKETS_SENT,
+    TIMESTAMP,
+)
 
 
 class Device:
     """Home Assistant representation of an UPnP/IGD."""
 
     def __init__(self, igd_device):
         """Initialize UPnP/IGD device."""
-        self._igd_device = igd_device
+        self._igd_device = igd_device  # type: IgdDevice
```

Comment:
We can use PEP 526 syntax from Python 3.6. https://www.python.org/dev/peps/pep-0526/
```suggestion
        self._igd_device: IgdDevice = igd_device
```